packages/ui/src/kit/VFileDropArea/VFileDropArea.vue

```diff
@@ -10,6 +10,11 @@
       class="v-file-drop-area-content"
       :class="themeCssClass"
     >
+      <file-paste-atom
+        v-if="isUseClipboard"
+        :disabled="disabled"
```

Comment:
I think, we can just not render this component if it is disabled and it should make `paste-atom` simplier